.github/workflows/ci.yaml

```diff
@@ -13,8 +13,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp-version: [23.1.3]
-        elixir-version: [1.11.2]
+        otp-version: [23.1.5, 23.2.7]
```

Comment:
Does it mean that it will run against these 2 versions?

Also, we target `23.2.7` here but exposed `23.2.6` in the tool-version below. Which one is good?

Reply:
It should be `23.2.7` everywhere…

Reply:
Fixed 👌🏻